apache/superset
# chore: Enhance Omnibar

### SUMMARY
This PR:

- Makes the Omnibar appear on the Dashboards' list with _cmd + k_ or _ctrl + k_
- Clears up the search when the Omnibar is closed
- Disables autocomplete
- Closes the Omnibar when clicking outside

https://user-images.githubusercontent.com/60598000/129581716-efc4100b-2156-4db6-bac6-1a9e68d25c2c.mp4

### TESTING INSTRUCTIONS
1. Open the dashboards' list
2. Press _cmd + k_ or _ctrl + k_ on your keyboard
3. Search for dashboards and move with the arrows through the results
4. Close the Omnibar by pressing ESC
5. Reopen the modal and observe that the latest search input and options are gone

### ADDITIONAL INFORMATION


- [x] Has associated issue: #16198
